docs/02-app/02-api-reference/01-components/link.mdx

```diff
@@ -645,10 +645,10 @@ export default Posts
 
 For example, you can generate a list of links to the dynamic route `app/blog/[slug]/page.js`:
 
-```tsx filename="app/blog/page.tsx" switcher
+```tsx filename="app/blog/[slug]/page.tsx" switcher
```

Comment:
This part seemed valid as is since you would create the list of links to sub-blog posts on the blog root page.

Reply:
Thank you. I've fixed it.